autobahn/wamp/interfaces.py

```diff
@@ -262,7 +262,10 @@ class ISession(object):
     @abc.abstractmethod
     def onConnect(self):
         """
-        Callback fired when the transport this session will run over has been established.
+        Callback fired when the transport this session will run over has
+        been established.
+
```

Comment:
Yes, splitting it up would be awesome in this case. The asynch callbacks probably right away - you've convinced me on that one already.